src/Bridge/Doctrine/Common/Filter/ContextAwareFilterInterface.php

```diff
@@ -26,5 +25,5 @@
     /**
      * Applies the filter.
      */
-    public function apply(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass, string $operationName = null, array $context = []);
+    public function apply($builder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass, string $operationName = null, array $context = []);
```

Comment:
why do we loose typing here?

Reply:
He explained it's because the QueryBuilder for the ODM isn't the same but he tried to reuse the code.

Reply:
Indeed, i understand. it is a good sign that this interface is missing in doctrine common, when we have an equal class in each backend implementation. Just for the naming :-)

Reply:
I suggest to revert this change and introduce a new interface, because QueryBuilder implementations aren't compatibles.